src/main/java/Doris.java

```diff
@@ -0,0 +1,134 @@
+import java.io.*;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.time.LocalDateTime;
+
+public class Doris {
+    ArrayList<Task> list = new ArrayList<Task>();
+
+    DateTimeFormatter df = DateTimeFormatter.ofPattern("yyyy-MM-dd hh:mm a");
```

Comment:
Perhaps use a more descriptive name for this? Its scope seems to be pretty large.